applications/launchpad/gui-react/src/components/TBot/TBotPrompt/index.tsx

```diff
@@ -170,51 +176,48 @@ const TBotPrompt = ({
   // Build messages list
   const renderedMessages = useMemo(() => {
     return messages?.slice(0, count).map((msg, idx) => {
+      const skipButtonCheck = count === idx + 1
       if (
         typeof msg !== 'string' &&
         typeof msg !== 'number' &&
         typeof msg !== 'boolean' &&
         msg
       ) {
+        // if message is complete functional component
```

Comment:
you don't need comments in situations like this, you can introduce a variable with meaningful name
(people read code, not comments :D)

```suggestion
        const componentMessage = 'content' in msg && typeof msg.content === 'function'
        if (componentMessage) {
```

Reply:
Getting back into more TypeScript issues with this. I've removed the comment.